src/app/pages/home/connect/connect.component.scss

```diff
@@ -4,10 +4,15 @@ ion-grid
     display: flex;
     flex-direction: column;
 
-    ion-row
-    {
-        flex: 1 0;
-    }
+    // ion-row
```

Comment:
Probably just remove the comment, we can always just add it in later if we need it